packages/frontend-app/src/app/components/Card/index.scss

```
@@ -0,0 +1,20 @@
+@import "tailwindcss/components";
+@import "tailwindcss/base";
+@import "tailwindcss/utilities";
+
+.card__container {
+  @apply flex flex-col cursor-pointer w-144 lg:w-156 xl:w-214 2xl:w-250;
+}
+
+.cover__image {
+  @apply rounded-md w-full max-h-144 lg:max-h-156 xl:max-h-214 2xl:max-h-250;
```

```suggestion
  @apply rounded-md w-full max-h-144 lg:max-h-156 xl:max-h-214 object-cover
```

We also wanna make sure that the image looks great within the card that's why we apply an `object-fit: cover`